src/system.js

```diff
@@ -37,12 +38,15 @@ module.exports = AFRAME.registerSystem('physics', {
 
     // If true, show wireframes around physics bodies.
     debug:                          { default: false },
+
+    // If using ammo, set the default rendering mode for debug
+    debugDrawMode: { default: THREE.AmmoDebugConstants.NoDebug }
   },
 
   /**
    * Initializes the physics system.
    */
-  init: function () {
+  async init() {
```

Comment:
Hm I'm not sure I'd introduce an async function here yet, although it would be very neat if A-Frame could allow components to tell it when they've initialized before going on to initialize components that depend on it. Looks like we can just use a callback or a promise here?

Reply:
In the case that the init does _not_ return a promise, `await this.driver.init({ ...` should act exactly the same as if the `await` wasn't there. Furthermore, aframe itself doesn't care that the system's init uses `async` or not, so this was the cleanest way I could allow ammo-driver to be async but not affect anything else. 
If you know of another way to do this that will work, I'm open to suggestions but of all of the different variations I've tried this seemed like the best given the constraints of aframe.

Reply:
👍